jobfair-frontend/src/containers/AttendantRegisterForm/AttendantRegisterForm.container.jsx

```diff
@@ -0,0 +1,73 @@
+import { Card, Form, Modal, Typography } from 'antd';
+import { PATH } from '../../constants/Paths/Path';
+import { registerAttendantAPI } from '../../services/jobhub-api/AttendantControllerService';
+import { useHistory } from 'react-router-dom';
+import AttendantRegisterFormComponent from '../../components/forms/AttendantRegisterForm/AttendantRegisterForm.component';
+import React from 'react';
+import RegisterSuccessContentComponent from '../../components/commons/RegisterSuccessContent/RegisterSuccessContent.component';
+
+const genderType = [
+  {
+    title: 'Male',
+    value: 'MALE'
+  },
+  {
+    title: 'Female',
+    value: 'FEMALE'
+  }
+];
+
+const AttendantRegisterFormContainer = () => {
+  const [form] = Form.useForm();
+  const history = useHistory();
+
+  const handleRegisterAttendant = async (values) => {
+    const body = {
+      ...values
+    };
+    try {
+      const res = await registerAttendantAPI(body);
+      if (res.status === 201) {
+        Modal.success({
+          title: 'Register attendant account successfully !',
+          width: '30rem',
+          closable: true,
+          keyboard: false,
+          maskClosable: true,
+          onOk: () => history.push(PATH.LOGIN_PAGE),
```

Comment:
use Href

Reply:
onOK used to specify a function that will be called when user clicks OK button. It's not a link, so I use history.push